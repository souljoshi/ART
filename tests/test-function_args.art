// checking arguments of function against global var

int x;

void foo(int x)
{
	printi(x);
	printc('\n');

	x = 10;

	printi(x);
	printc('\n');
}

int main()
{
	x = 5;
	printi(x);
	printc('\n');

	foo(x);

	printi(x);
	printc('\n');

	foo(20);
}


	
