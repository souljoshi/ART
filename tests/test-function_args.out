5
5
10
5
20
10
